### ISSUE TITLE
Help offered

### ISSUE THREAD

**Issue by jhjacobs81** [miroslavpejic85/mirotalksfu#21]
I could help with translations into Dutch and German

**Comment by miroslavpejic85** [comment]
Hi, But you can use Google Translate plugin and translate it in all language with a single click :)

<img width="1186" alt="translate" src="https://user-images.githubusercontent.com/70838434/136613478-cef159d1-c38c-45d9-8406-fd7552decdc9.png">

**Comment by jhjacobs81** [comment]
our users arent that clever ;)

**Comment by miroslavpejic85** [comment]
Yes replace English with Dutch and so on... but Google Translate add-ons is the fast way without touch anything ;)
and is free...
